.code-samples.meilisearch.yaml

```diff
@@ -1280,9 +1312,9 @@ delete_a_key_1: |-
 authorization_header_1:
   let client = Client::new("http://localhost:7700", "masterKey");
   let keys = client
-    .get_keys()
-    .await
-    .unwrap();
+  .get_keys()
+  .await
+  .unwrap();
```

Comment:
Not sure about this formatting. Is it not more common to indent the methods?

Reply:
I ran `cargo fmt` and the command gave this output. Is there something that I need to change here?

Reply:
If cargo fmt said so, lets keep it like that ! 🔥